src/main/java/pages/TestPage.java

```diff
@@ -180,5 +184,21 @@ public String getOdometerValue() {
         }
         return value;
     }
+
+    public boolean isTestNotCompleteDisplayed() {
+        return findElementByAccessibilityId(TEST_NOT_COMPLETE_TITLE).isDisplayed() && findElementByAccessibilityId(TEST_NOT_COMPLETE_INFO).isDisplayed();
+    }
+
+    public void clickReviewAndConfirm() {
+        findElementById(REVIEW_AND_CONFIRM_BUTTON_ID).click();
+    }
+
+    public boolean buttonOkIsClickable() {
```

Comment:
I think that if you check that is enabled the isDisplayed check becomes redundantDictionary

Reply:
from what i know isEnabled does not exclude isDisplayed. I will leave it as it is.